src/main/java/teammates/common/datatransfer/FeedbackSessionState.java

```diff
@@ -0,0 +1,5 @@
+package teammates.common.datatransfer;
+
+public enum FeedbackSessionState {
+    OPEN, CLOSED, PUBLISHED;
+}
```

Comment:
This enum is not defined yet. We use strings from Const.java. 
There is a FeedbackSessionType enum which has STANDARD and PRIVATE.